From 0aab4c61cf62278f46478a59414845d06fb3c0b0 Mon Sep 17 00:00:00 2001
From: Nachiket Sahasrabuddhe <nachiket_sahasrabuddhe@xyratex.com>
Date: Mon, 17 Feb 2014 14:16:52 +0530
Subject: [PATCH 13/14] dgmode-spare-map-writeIO

-Removed symantically incorrect line from the code. It was left there
 as code was not compiled after last modifications.
---
 m0t1fs/linux_kernel/file.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/m0t1fs/linux_kernel/file.c b/m0t1fs/linux_kernel/file.c
index 85fb6b4..ef8451c 100644
--- a/m0t1fs/linux_kernel/file.c
+++ b/m0t1fs/linux_kernel/file.c
@@ -3159,7 +3159,6 @@ static int ioreq_dgmode_write(struct io_request *req, bool rmw)
 	 * IO request will be reshuffled by redirecting pages meant for
 	 * failed device(s) to its corresponding spare unit(s).
 	 */
-	       req->ir_rc, req->ir_nwxfer.nxr_rc);
 
 	/*
 	 * Finalizes current fops which are not valid anymore.
-- 
1.8.3.2

